style.css

```diff
@@ -131,8 +131,92 @@ a {
   text-decoration: underline;
 }
 
+/* wrapper for side navbar and component display section  */
+.wrapper {
+  width: 90%;
+  height: 100vh;
+  margin: 0 auto;
+  
+}
+
+.component-container {
+  display: flex;
+}
+
+ /* Side Navbar */
+
+ 
+
+.sidenav {
+  flex-basis: 20%;
+  padding: 0.8rem;
+  margin-top: 0.8rem;
+}
+.sidenav-heading {
+  padding-bottom: 0.8rem;
+}
+
+.sidenav li {
+  list-style-type: none;
+  padding: 0.8rem;
+
+  margin-bottom: 0.8rem;
+}
+
+.sidenav li a:hover {
+  color: var(--lynx-white-color);
+  background-color: var(--secondary-color);
+}
+
+.active {
+  color: var(--secondary-color);
+  font-weight: var(--bold-font);
+}
+
+ /* component display section  */
+
+.comp-display-section {
+  width: 70%;
+  margin: 0 auto;
+  margin-top: 0.8rem;
```

Comment:
You could use the 3 valued shorthand for this purpose as it does the same work in less lines.

`margin: 0.8rem auto 0;` 

which works in this way
`margin: [margin-top] [margin-left-right] [margin-bottom];`

The third demo in [this](https://developer.mozilla.org/en-US/docs/Web/CSS/margin) might help visualize

Reply:
Thank you for this explanation. I am gonna use this in the future.